be/test/exprs/string_functions_test.cpp

```diff
@@ -0,0 +1,131 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "exprs/string_functions.h"
+#include "util/logging.h"
+#include "exprs/anyval_util.h"
+#include <iostream>
+#include <string>
+
+#include <gtest/gtest.h>
+
+namespace doris {
+
+    class StringFunctionsTest : public testing::Test {
```

Comment:
there is no indent here. It is different with Java

Reply:
OK